src/utils/socket.js

```diff
@@ -0,0 +1,50 @@
+const app = require("../app");
+const http = require("http");
+const socketIO = require("socket.io");
+
+const server = http.createServer(app);
+
+server.listen(process.env.PORT || 4000, () => {});
+
+const io = socketIO(server, {
+  cors: {
+    origin: "http://localhost:3000",
+  },
+});
```

Comment:
```suggestion
const io = socketIO(server, {
  cors: {
    origin: "http://localhost:3000",
    methods: ["GET", "POST"]
  },
});
```
methods를 추가해줘야 할 것 같습니다 😄

Reply:
추가 하겠습니다!